examples/postprocessing/index.js

```diff
@@ -18,7 +18,7 @@ dragon.positions = centerAndNormalize(dragon.positions).map((v) => vec3.scale(v,
 dragon.normals = normals(dragon.cells, dragon.positions)
 dragon.uvs = dragon.positions.map(() => [0, 0])
 const parseHdr = require('parse-hdr')
-const ctx = createContext()
+const ctx = createContext({ precision: 'mediump' })
```

Comment:
Added just to test it was working when using a lower precision even with a higher one available.